CardScan/Classes/OcrDD.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 
 
 public class OcrDD{
+    public var scanStats = ScanStats()
```

Comment:
instead of using scanstats for this why not create a variable that's just for a list of boxes?

Reply:
This is consistent with how Ocr.swift does it, incase we want to add more stuff to this. But I can create a variable specific to this if you prefer.

Reply:
Yeah, OCR using scanstats is a bit of a throwback to the old way of doing it -- we track the real scan stats in the main loop class. Let's go with a specific variable.

Reply:
Alright.